src/main/resources/edu/cornell/kfs/pmw/cu-ojb-pmw.xml

```diff
@@ -0,0 +1,172 @@
+<!--
+   - The Kuali Financial System, a comprehensive financial management system for higher education.
+   - 
+   - Copyright 2005-2014 The Kuali Foundation
+   - 
+   - This program is free software: you can redistribute it and/or modify
+   - it under the terms of the GNU Affero General Public License as
+   - published by the Free Software Foundation, either version 3 of the
+   - License, or (at your option) any later version.
+   - 
+   - This program is distributed in the hope that it will be useful,
+   - but WITHOUT ANY WARRANTY; without even the implied warranty of
+   - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   - GNU Affero General Public License for more details.
+   - 
+   - You should have received a copy of the GNU Affero General Public License
+   - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ -->
+<descriptor-repository version="1.0">
+
+<class-descriptor class="edu.cornell.kfs.pmw.batch.businessobject.PaymentWorksVendor" table="CU_PMW_VENDOR_T">
+    <field-descriptor name="pmwVendorRequestId" column="PMW_VND_REQ_ID" jdbc-type="VARCHAR" primarykey="true" />
+    <field-descriptor name="versionNumber" column="VER_NBR" jdbc-type="BIGINT" locking="true" />
+    <field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" index="true" />
+    
+    <field-descriptor name="pmwRequestStatus" column="PMW_REQ_STAT" jdbc-type="VARCHAR" />
+    <field-descriptor name="pmwTransactionType" column="PMW_TRANS_CD" jdbc-type="VARCHAR" />
+
+    <field-descriptor name="kfsVendorProcessingStatus" column="KFS_VND_PROC_STAT" jdbc-type="VARCHAR" />
+    <field-descriptor name="kfsVendorDocumentNumber" column="PVEN_FDOC_NBR" jdbc-type="VARCHAR" />
+    <field-descriptor name="kfsVendorHeaderGeneratedIdentifier" column="VNDR_HDR_GNRTD_ID" jdbc-type="INTEGER" />
+    <field-descriptor name="kfsVendorDetailAssignedIdentifier" column="VNDR_DTL_ASND_ID" jdbc-type="INTEGER" />
+    <field-descriptor name="kfsAchDocumentNumber" column="PAAT_FDOC_NBR" jdbc-type="VARCHAR" />
+    <field-descriptor name="kfsAchProcessingStatus" column="KFS_ACH_PROC_STAT" jdbc-type="VARCHAR" />\
+    <field-descriptor name="processTimestamp" column="PROC_TS" jdbc-type="TIMESTAMP"/>
+
+    <field-descriptor name="requestingCompanyId" column="REQ_CMP_ID" jdbc-type="VARCHAR" />
```

Comment:
I wonder if it would have made sense for this to be a separate BO

Reply:
Per our offline conversation, this will not be broken into another BO as this data is all part of the single PMW form submitted by a single vendor.  Breaking it into another business object would make tracking the data submitted more difficult.